packages/scanner/src/report/scanSummary.ts

```diff
@@ -20,3 +22,41 @@ export interface ScanSummary {
   numFindings: number;
   appMapMetadata: AppMapMetadata;
 }
+
+export function aggregateScanSummary(
+  sourceScanSummary: ScanSummary,
+  targetScanSummary: ScanSummary
+) {
+  targetScanSummary.numAppMaps += sourceScanSummary.numAppMaps;
+  targetScanSummary.numChecks += sourceScanSummary.numChecks;
+  sourceScanSummary.rules.forEach((value) => targetScanSummary.rules.push(value));
+  sourceScanSummary.ruleLabels.forEach((value) => targetScanSummary.ruleLabels.push(value));
+  targetScanSummary.numFindings += sourceScanSummary.numFindings;
+  sourceScanSummary.appMapMetadata.labels.forEach((value) =>
+    targetScanSummary.appMapMetadata.labels.push(value)
+  );
+  sourceScanSummary.appMapMetadata.apps.forEach((value) =>
+    targetScanSummary.appMapMetadata.apps.push(value)
+  );
+  sourceScanSummary.appMapMetadata.clients.forEach((value) =>
+    targetScanSummary.appMapMetadata.clients.push(value)
+  );
+  sourceScanSummary.appMapMetadata.frameworks.forEach((value) =>
+    targetScanSummary.appMapMetadata.frameworks.push(value)
+  );
+  sourceScanSummary.appMapMetadata.git.forEach((value) =>
+    targetScanSummary.appMapMetadata.git.push(value)
+  );
+  sourceScanSummary.appMapMetadata.languages.forEach((value) =>
+    targetScanSummary.appMapMetadata.languages.push(value)
+  );
+  sourceScanSummary.appMapMetadata.recorders.forEach((value) =>
+    targetScanSummary.appMapMetadata.recorders.push(value)
+  );
+  sourceScanSummary.appMapMetadata.testStatuses.forEach((value) =>
+    targetScanSummary.appMapMetadata.testStatuses.push(value)
+  );
+  sourceScanSummary.appMapMetadata.exceptions.forEach((value) =>
+    targetScanSummary.appMapMetadata.exceptions.push(value)
+  );
```

Comment:
I don't think it's worth capturing all of this AppMap metadata, especially without de-duplicating elements.

Reply:
I went ahead and de-duplicated them.  I don't mind not aggregating them at all and sending a blank array.